src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionServiceTest.kt

```diff
@@ -225,67 +165,6 @@ class SessionServiceTest {
       assertDate(sessions[0].startTimestamp, "2021-02-01T11:30:00", MONDAY)
     }
 
-    @Test
-    fun `all sessions are on past dates, no sessions are returned`() {
```

Comment:
Is this no longer a valid test

Reply:
not sure, I think I might have to still test for this!